subsys/bluetooth/controller/hal/nrf5/radio.c

```diff
@@ -472,11 +472,12 @@ void radio_switch_complete_and_rx(u8_t phy)
 #else /* !CONFIG_BLUETOOTH_CONTROLLER_TIFS_HW */
 	NRF_RADIO->SHORTS = RADIO_SHORTS_READY_START_Msk |
 			    RADIO_SHORTS_END_DISABLE_Msk;
-	sw_switch(0, phy, 0);
+	sw_switch(0, 0, 0, phy_rx, 0);
 #endif /* !CONFIG_BLUETOOTH_CONTROLLER_TIFS_HW */
 }
 
-void radio_switch_complete_and_tx(u8_t phy, u8_t flags)
+void radio_switch_complete_and_tx(u8_t phy_rx, u8_t flags_rx, u8_t phy_tx,
+				  u8_t flags_tx)
```

Comment:
these alignments are github diff displaying so, due to previous line having a + prefixed.